EF/DeliveryService/DeliveryService/Models/Address.cs

```diff
@@ -0,0 +1,28 @@
+﻿using DeliveryService.Models.BaseModel;
+
+namespace DeliveryService.Models
+{
+    public class Address : Model
```

Comment:
please remove all methods from models
usually it is about data structure 
not about behavior 
so no need methods